sessionctx/variable/session.go

```diff
@@ -407,8 +407,8 @@ func (s *SessionVars) GetNextPreparedStmtID() uint32 {
 	return s.preparedStmtID
 }
 
-// GetTimeZone returns the value of time_zone session variable.
-func (s *SessionVars) GetTimeZone() *time.Location {
+// Location returns the value of time_zone session variable. If it is nil, then return time.Local.
+func (s *SessionVars) Location() *time.Location {
 	loc := s.TimeZone
```

Comment:
We may need to rename `s.TimeZone` to `s.Location` or any other proper names.

Reply:
Location is better and also time package uses the same name.

Reply:
Yes, we can do it in another PR, keep this PR focused on the daylight saving time issue.

Reply:
Already did - -.

Reply:
I changed to `Loc` eventually.

Reply:
OK

Reply:
I saw a conflict to avoid further change I undo my changes and I will file another PR for changing this once this PR gets merged into master.